src/main/scala/ProduceRoadMask.scala

```diff
@@ -0,0 +1,55 @@
+package geotrellis.sdg
+
+import geotrellis.proj4._
+import geotrellis.spark.store.kryo._
+import geotrellis.layer._
+
+import org.locationtech.geomesa.spark.jts._
+
+import org.apache.spark.SparkConf
+import org.apache.spark.serializer.KryoSerializer
+import org.apache.spark.sql._
+
+import cats.implicits._
+import com.monovore.decline._
+
+
+object ProduceRoadMask extends CommandApp(
+  name = "Road Masking",
+  header = "Poduces road geometries from OSM as an orc file",
+  main = {
+    val mbtilesFile = Opts.option[String]("input", help = "The path to the orc file that should be read")
+    val country = Opts.option[String]("country", help = "The Alpha-3 code for a country from the ISO 3166 standard")
```

Comment:
While I'm not a fan of having to pass in the country code directly, it was the quickest way in which I could assure that the geometries were assigned a country correctly. This is a bit limiting, though, as it assumes that the given `mbtiles` file contains a single country. One possible workaround is to be able to pass in multiple `mbtiles` and countries, but I'm not sure if that's ideal.